### ISSUE TITLE
M11 inventory picture not found

### ISSUE THREAD

**Issue by jonpas** [toadie2k/NIArms#159]
![image](https://user-images.githubusercontent.com/7935003/59152447-6d705b80-8a44-11e9-9782-6fdf70595399.png)

**Comment by Blutze** [comment]
Confirmed with CBA 3.11.2 and SWS AiO, no other mods.

Its weird though... that file isn't referenced anywhere in the config. Not on the gun, not on the JR slots, not in the (de-pbo'd as well as on GitHub) 226 or Core code. Arsenal icon also works properly (otherwise this would have been noticed much earlier).

There would be the Trijicon XS sights with a very similar filename. But they don't have any issues.

Error message is posted twice in the .rpt, for whatever that is worth.

**Comment by jonpas** [comment]
Yes, weird, I assume Toadie's local files are not the same as here.

**Comment by robalo** [comment]
Can be avoided by conforming to inventory icons naming standards: https://community.bistudio.com/wiki/Arma_2_Weapon_to_Arma_3#Inventory_Icons
